src/app/shared/shared.module.ts

```diff
@@ -1,8 +1,6 @@
-/* @format */
 import { NgModule, ComponentFactoryResolver } from '@angular/core';
```

Comment:
Ideally this file should have no changes to it at all, but especially including getting rid of the format pragma for Prettier.

Reply:
Same as before regarding the pragma @meisekimiu